src/Common/Core/Impl/Microsoft.Common.Core.csproj

```diff
@@ -42,6 +42,7 @@
     <Reference Include="System.IO.Compression.FileSystem" />
     <Reference Include="System.Security" />
     <Reference Include="System.Windows.Forms" />
+    <Reference Include="WindowsBase" />
```

Comment:
It is for the AF below `Dispatcher.FromThread` I guess we could add some sort of 'has shutdown sharted' to `ICoreShell`